packages/host/tests/integration/components/operator-mode-test.gts

```diff
@@ -2265,6 +2265,117 @@ module('Integration | operator-mode', function (hooks) {
+              'This is the first message replacement comes after second message, but must be displayed before second message because it will be used creted from the oldest',
+            format: 'org.matrix.custom.html',
+          },
+        },
+        origin_server_ts: new Date(2024, 0, 3, 12, 30).getTime(),
+        unsigned: {
+          age: 105,
+          transaction_id: '1',
+        },
+        status: null,
+      };
+      let secondMessage = {
+        event_id: 'second-message-event',
+        room_id: roomId,
+        state_key: 'state',
+        type: 'm.room.message',
+        sender: '@aibot:localhost',
+        content: {
+          body: 'This is the second message comes after the first message and before the replacement of the first message',
+          msgtype: 'org.text',
+          formatted_body:
+            'This is the second message comes after the first message and before the replacement of the first message',
+          format: 'org.matrix.custom.html',
+        },
+        origin_server_ts: new Date(2024, 0, 3, 12, 31).getTime(),
+        unsigned: {
+          age: 105,
+          transaction_id: '1',
+        },
+        status: null,
+      };
+      let firstMessageReplacement = {
+        event_id: 'first-message-replacement-event',
+        room_id: roomId,
+        state_key: 'state',
+        type: 'm.room.message',
+        sender: '@aibot:localhost',
+        content: {
+          body: 'This is the first message replacement comes after second message, but must be displayed before second message because it will be used creted from the oldest',
+          msgtype: 'org.text',
+          formatted_body:
+            'This is the first message replacement comes after second message, but must be displayed before second message because it will be used creted from the oldest',
+          format: 'org.matrix.custom.html',
+          ['m.new_content']: {
+            body: 'This is the first message replacement comes after second message, but must be displayed before second message because it will be used creted from the oldest',
+            msgtype: 'org.text',
+            formatted_body:
+              'This is the first message replacement comes after second message, but must be displayed before second message because it will be used creted from the oldest',
+            format: 'org.matrix.custom.html',
+          },
+          ['m.relates_to']: {
+            event_id: 'first-message-event',
+            rel_type: 'm.replace',
+          },
+        },
+        origin_server_ts: new Date(2024, 0, 3, 12, 32).getTime(),
+        unsigned: {
+          age: 105,
+          transaction_id: '1',
+        },
+        status: null,
+      };
+
+      await addRoomEvent(matrixService, firstMessage);
+
+      await addRoomEvent(matrixService, secondMessage);
+
+      await addRoomEvent(matrixService, firstMessageReplacement);
+
+      await waitFor('[data-test-message-idx="0"]');
+      await this.pauseTest();
```

Comment:
This should be removed, I wonder if we could lint against it 🧐

Reply:
It seems I have to open another PR to fix the linter issue. I found that the current `eslint-plugin-ember` version that we used doesn't do linter check for the test file with `.gts` extension. I've tried to update it to the latest version, it worked but it caused many linter errors, so I'll resolve it in another PR.